tests/helpers/permissions.ts

```diff
@@ -0,0 +1,51 @@
+import { LxdVersions } from "../fixtures/lxd-test";
+import { test, expect } from "../fixtures/lxd-test";
+import { Page } from "@playwright/test";
+
+export const skipIfNotSupported = (lxdVersion: LxdVersions) =>
+  test.skip(
+    lxdVersion === "5.0-edge",
+    "Fine grained authorisation is not available for lxd 5.0",
+  );
+
+export const assertModificationStatus = async (
+  page: Page,
+  statusText: string,
+) => {
+  const status = page.getByText(statusText);
+  await expect(status).toBeVisible();
+};
```

Comment:
```suggestion
export const assertTextVisible = async (
  page: Page,
  statusText: string,
) => {
  const status = page.getByText(statusText);
  await expect(status).toBeVisible();
};
```
This seems like a useful function. Shall we move it into a more generic helper file and adopt it in other tests as well? Could be a follow-up task. We can use this instead of the current approach: In many tests, we currently click on the text instead of using expect. Maybe create a ticket for this?

Reply:
Good idea :+1: here is the [issue](https://warthogs.atlassian.net/browse/WD-11114)